packages/page-bounties/src/Voting/VotingLink.tsx

```diff
@@ -10,7 +10,7 @@ function VotingLink (): React.ReactElement<string> {
   const { t } = useTranslation();
 
   return (
-    <div>
+    <div style={{ margin: ' 0 1rem 0 0' }}>
```

Comment:
Would really prefer not to do this. This is a nightmare to maintain.

Reply:
ok, fixed :)